### ISSUE TITLE
Split view of same file: opening file is not intuitive

### ISSUE THREAD

**Issue by derekatkinson** [adobe/brackets#12109]
I was excited to try out the new "split view of same file" feature in 1.6, but after looking high and low could not figure out how to do it and had to find the answer in Github comments: https://github.com/adobe/brackets/pull/11820#issuecomment-173275272

It would be more intuitive to have a global menu item or an "Open in other pane" item in the menu when right-clicking on the name of an open file in the file pane.

**Comment by dakaraphi** [comment]
Same here.  Initially I couldn't figure out how to do it.  When you have a huge file tree, you would want to be able to have a menu action on the current open files and not have to find it in the file tree.

**Comment by JKAussieSkater** [comment]
Agreed! For reference, the comment which explains how to do this is:

> [<img src="https://avatars2.githubusercontent.com/u/7641760?v=3&s=96" width="45px" align="left">](https://github.com/petetnt) __[petetnt](https://github.com/petetnt)__ commented
> `@depiction` Open Split View and open a file to the first pane. Then focus the other pane and open the same file again (from the file tree). You should now see the same file on both panes.
